backend/src/main/repositories/EmployeeAccountRepository.ts

```diff
@@ -108,4 +108,22 @@ export default class EmployeeAccountRepository implements CRUD {
       return Promise.reject(err);
     }
   };
+
+  public getAllByBusiness = async (businessEmail: string): Promise<EmployeeAccount[]> => {
+    try {
+      const clientAccounts = await EmployeeAccount.findAll({
+        where: {
+          supervisorEmail: businessEmail,
+        },
+      });
+
+      log(`retrieved all employee accounts`);
+      return Promise.resolve(clientAccounts);
+    } catch (err: any) {
+      if (err instanceof BaseError) {
+        throw new Error(`${err.name}, message: ${err.message}`);
```

Comment:
I think you should `Promise.reject( new Error......)` instead of throw. Cause it might not be caught because of async. https://stackoverflow.com/questions/33445415/javascript-promises-reject-vs-throw#:~:text=Any%20time%20you%20are%20inside,callback%2C%20you%20must%20use%20reject%20.